### ISSUE TITLE
Ability to disable notifications for specific channels

### ISSUE THREAD

**Issue by Zach443** [hexchat/hexchat#1172]
It would be amazing if you guys added the ability to silence notifications for certain channels. I (and more than likely others too), am in a channel where my username gets used frequently, however I can not turn off notifications entirely because there are other important channels that I need to be aware of.

**Comment by ehasting** [comment]
I agree - in a context where i am on several network, where some channels are more for lingering during workhours, disabeling notification for the network or channels would be awesome.

**Comment by timstoop** [comment]
+1 This would be really great.

**Comment by mildred** [comment]
In the same context, it would be nice to be able to configure the level of notification for each channel. I'm using Slack for my work, and it has this feature. It's very nice to be able for each channel to:

- silence it entirely
- be notified each time you are getting mentioned
- be notified for each and every message

Also, it has a snooze feature that lets you delay notifications for a configurable amount of time. notifications resume afterwards.

**Comment by mB-PiBox** [comment]
Completely possible via scripting
Good place to start https://github.com/TingPing/plugins/blob/master/HexChat/highlight.lua
(here's the .py if you not familiar with lua -- https://github.com/TingPing/plugins/blob/master/HexChat/duplicates/highlight.py)

That will show you how to hook the highlight, then modify to check channel name, and receive user code "/highlight off #channel" if highlight matches #channel then hexchat.EAT_ALL

**Comment by chefarov** [comment]
Per channel notifications are essential for those who join several channels. I would appreciate it too :)
Thanks for the scripting workaround @mB-PiBox.

**Comment by jwgn** [comment]
+1 Please implement this feature.

**Comment by Belatar** [comment]
+1
